model/things/src/test/java/org/eclipse/ditto/model/things/NullAttributesTest.java

```diff
@@ -61,14 +60,26 @@ public void nullAttributesIsJsonNull() {
         assertThat(underTest.isNull()).isTrue();
     }
 
+    @Test
+    public void nullAttributesIsJsonObject() {
+        assertThat(underTest.isObject()).isTrue();
+    }
+
+    @Test
+    public void nullAttributesAsJsonObject() {
+        assertThat(underTest.asObject()).isEqualTo(JsonFactory.nullObject());
```

Comment:
Wouldn't it rather be ... `isEqualTo(underTest)` or even `isSameAs(underTest)`?

Reply:
Depends on what we want to test. I just wanted to make sure that `asObject()` returns a null object.